easybuild/easyconfigs/g/GoPeaks/GoPeaks-1.0.0.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'GoPackage'
+
+name = 'GoPeaks'
+version = '1.0.0'
+
+homepage = 'https://git-lfs.github.com'
+description = """GoPeaks is a peak caller designed for CUT&TAG/CUT&RUN sequencing data. GoPeaks by default works best
+ with narrow peaks such as H3K4me3 and transcription factors. However, broad epigenetic marks like H3K27Ac/H3K4me1
+ require different the step, slide, and minwidth parameters. We encourage users to explore the parameters of GoPeaks
```

Comment:
I'm sure this is copied from upstream, but perhaps it could be corrected (let them know too).